File: Modul-3/Soal-4/PRAK304-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(){
    int bil;
    printf("Masukkan bilangan: ");
    scanf("%d", &bil);

    if (bil == 0)
    {
        printf("Nol");
    }else if (bil >= 1 && bil <= 9)
    {
        printf("Satuan");
    }else if (bil == 10 || bil >= 20 && bil <= 99)
    {
        printf("Puluhan");
    }else if (bil >= 11 && bil <= 19)
    {
        printf("Belasan");
    }else
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    
    return 0;
}

File: Modul-3/Soal-4/PRAK304-2410817220012-NOOR-KHALISA.py
bil = int(input("Masukkan bilangan: "))

if bil == 0: print("Nol")
elif 0 < bil < 10: print("Satuan")
elif bil == 10 or (20 <= bil <= 99): print("Puluhan")
elif bil <= bil <= 19: print("Belasan")
else: print("Anda Menginput Melebihi Limit Bilangan")